python/lancedb/table.py

```diff
@@ -124,7 +131,8 @@ def add(self, data: DATA, mode: str = "append") -> int:
         """
         data = _sanitize_data(data, self.schema)
         ds = lance.write_dataset(data, self._dataset_uri, mode=mode)
-        return ds.count_rows()
+        self._reset_dataset()
+        len(self)
```

Comment:
this does not return anything, right? (this is python)

Reply:
yeah my mistake